### ISSUE TITLE
0.13.16 tgz

### ISSUE THREAD

**Issue by NeQuissimus** [sbt/sbt#3384]
Could you release a tgz for 0.13.16 into Bintray (https://dl.bintray.com/sbt/native-packages/sbt/) as you did with other versions?

Or is there a new place for these to show up?

Thank you

**Comment by eed3si9n** [comment]
We are working on another 1.0.0-RCx, so I wanted to announce both together.
sbt 0.13.16 tgz link is up on http://www.scala-sbt.org/download.html

**Comment by bpholt** [comment]
FWIW, the link on http://www.scala-sbt.org/download.html isn't ideal for a couple reasons:

1. It's not predictable. We build [Docker images](https://github.com/Dwolla/jenkins-agent-docker-sbt) for our Jenkins instances to use for building sbt projects. Using bintray (or some predictable scheme) allows us to easily parameterize the sbt version that should be included in the Docker image. I discovered this issue because I tried to naively update the image this morning to 0.13.16 and the build failed. 🙁
1. It redirects to an HTTP URL, which is insecure. We strongly prefer to obtain libraries and build tools via HTTPS.
